build.gradle

```diff
@@ -26,28 +26,35 @@ ext.ref = parseprops(file('build.properties'))
 ext.ref.mod_deps = parseprops(file('mod_deps.properties'))
 
 group = ref.package_base + '.' + ref.mod_id
-archivesBaseName = ref.mod_abbr
-// remove travis stuff from version for now because we aren't using it and it was a mess. -srs
-version = ref.mc_version + '-' + ref.mod_version
+archivesBaseName = (ref.mod_id as String).toUpperCase() + '-' + (ref.mc_version as String)
+version = ref.mod_version
+
+repositories {
+    maven { url "http://files.minecraftforge.net/maven" } //BiomesOPlenty
```

Comment:
Dammit, that was supposed to be part of this commit. Will add it in a bit, so don't merge it yet.